sktime_dl/deeplearning/resnet/_regressor.py

```diff
@@ -0,0 +1,132 @@
+__author__ = "James Large, Withington"
+
+import keras
+import numpy as np
+import pandas as pd
+
+from sklearn.base import RegressorMixin
+
+from sktime.utils.validation.supervised import validate_X, validate_X_y
+
+from sktime_dl.deeplearning.base.estimators import BaseDeepRegressor
+from sktime_dl.deeplearning.resnet._base import ResNetNetwork
+
+
+class ResNetRegressor(BaseDeepRegressor, RegressorMixin, ResNetNetwork):
```

Comment:
Same here: `BaseDeepRegressor` already inherits from `RegressorMixin`, perhaps quickly search and replace throughout the repo.

Reply:
Good spot, thanks.